script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -233,12 +233,12 @@ As well as the 128 Cardinals assisting him.
 [sha:3564f7b08ec911567734f56dc35c392a78c7b9a0]{
 -- Page 804, Offset 13031.
 -- 最高権力者たちの集まりである枢機卿の中でも最大の発言力を持ち、次代の法王は確実と言われているのがラウレンティス枢機卿であり、その孫がこの少年、マーリオゥなのである。
-Even amongst those who hold the highest of power, Cardinal Laurentis is certainly said to be the most influential and the next in line to become Pope. His grandson is Mario, this present boy.
+Even amongst those in the highest positions of power, Cardinal Laurentis is said to be the most influential, and the next in line to become Pope. His grandson is Mario, this boy.
```

Comment:
Even among those who hold the highest positions of power, Cardinal Laurentis is said to be the most influential. He's the next in line to become pope.